### PR TITLE
Cron improvements

### PR DESCRIPTION
**Pull request by mikaukora**
- Split harvester cron job into index specific jobs, "location" and "event".
- Fix address resolving. Variable names must match exactly on Python and TS side.
- Allow HTTP errors when storing Linkedevents place. In some cases the place is not available or there's NW error. Just skip and continue processing other units.

### CODE REVIEW

File: sources/ingest/fetch/location.py [hunk 1]
```diff
@@ -351,15 +352,19 @@ def fetch():
...
+                service="linkedevents",
+                origin_url=place_url,
+                raw_data=place)
 
-        root = Root(venue=venue)
-        root.links.append(place_link)
+            root = Root(venue=venue)
+            root.links.append(place_link)
+        except (requests.exceptions.HTTPError, requests.exceptions.ConnectionError):
+            logger.warning(f"Error when fetching {place_url}")
```
Comment (tuomas777) [inline sources/ingest/fetch/location.py:366]: Maybe also log the actual error here to ease problem-solving?

Comment (mikaukora) [reply sources/ingest/fetch/location.py]: Good idea, fixing.

File: sources/ingest/fetch/location.py [hunk 2]
```diff
@@ -351,15 +352,19 @@ def fetch():
...
+                origin_url=place_url,
+                raw_data=place)
 
-        root = Root(venue=venue)
-        root.links.append(place_link)
+            root = Root(venue=venue)
+            root.links.append(place_link)
+        except (requests.exceptions.HTTPError, requests.exceptions.ConnectionError) as exc:
+            logger.warning(f"Error while fetching {place_url}: {exc}")
+            pass
```
Comment (tuomas777) [inline sources/ingest/fetch/location.py:367]: Unnecessary `pass` 🙂

**Review by tuomas777 — APPROVED** [review]
LGTM 👍 🚀

File: sources/ingest/fetch/location.py (hunk 2 above)
Comment (mikaukora) [reply sources/ingest/fetch/location.py]: Fixing :)

**Pull request merged**